package Programs;

public class JavaExceptions {

	public static void main(String[] args) 
	{
		System.out.println("Start");
		try
		{
			int divide =10/0;
			 System.out.println(divide);
			 
		}
		catch (Exception e)
		{
			System.out.println("Error occured");
		}
		System.out.println("End");
	}

}
